SomosMasApp/SomosMasApp/App/Shared/Modules/LogIn/ViewModel/LogInViewModel.swift

```diff
@@ -54,4 +56,45 @@ class LogInViewModel {
         return (result,resultMessage)
     }
     
+    // [OT171-25] Save Token with UserDefaults:
+    func sessionIsSaved(user: Credentials) {
+        LoginService().login(user: user) { token in
+            // Token Saved
+            let session = UserSession.buildWithToken(token)
+            self.newUserSession(session)
+            // Navigation to Home View here
+        } errorHandler: { errorMessage in
+            // Modal for Error could be here.
+        }
+    }
+}
+
+struct UserSession {
+    
+    static var tokenIsSaved: UserSession? {
+        let userDefaults = UserDefaults.standard
+        guard let token = userDefaults.object(forKey: "token") as? String else {
+            // In case of have no token saved:
+            return nil
+        }
+        return UserSession(token: token)
+    }
+    
+    var token: String {
+        let userDefaults = UserDefaults.standard
```

Comment:
this is ok but please remove it so an other team member create this when it need it, no problem to do more but if you are already correcting.....